src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console */
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
```

Comment:
Your require to do the following concerning your check status button.

Define CHECK_STATUS as action type (name can be sligthtly different).

The value of  action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:       const CHECK_STATUS = bookstore/categories/CHECK_STATUS



Sets an initial state with an empty array.
Creates and Exports Action Creator. (checkStatus function)
Creates a reducer and exports it as default.

in case action.type is CHECK_STATUS the function returns a string "Under construction".
if the action.type is different, it returns the state with no changes.